statements.php

```diff
@@ -0,0 +1,110 @@
+<?php
```

Comment:
Each file should start with why it exists. That is, explain what types of functions live in that file and why you would call them.